### PR TITLE
add sentry

### PR DESCRIPTION
**Pull request by Anakin100100**
#48

### CODE REVIEW

File: config/initializers/sentry.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Sentry.init do |config|
+    # config.dsn is deliberately not configured.
+    # The dsn will be collected fron SENTRY_DSN env variable
+    
+    config.breadcrumbs_logger = [:active_support_logger, :http_logger]
+
+    # To activate performance monitoring, set one of these options.
+    # We recommend adjusting the value in production:
+    config.traces_sample_rate = 1
```
Comment (micorix) [inline config/initializers/sentry.rb:9]: Pls make it customizable by env var

**Review by micorix — CHANGES_REQUESTED** [review]

File: config/initializers/sentry.rb (hunk 1 above)
Comment (Anakin100100) [reply config/initializers/sentry.rb:9]: It is customizable already as per sentry docs

What the DSN Does

A DSN tells a Sentry SDK where to send events so the events are associated with the correct project.

If this value is not provided, SDKs will try to read it from the SENTRY_DSN environment variable, where applicable. This fallback does not apply in cases like a web browser, where the concept of environment variables does not exist.

If an SDK is not initialized or if it is initialized with an empty DSN, the SDK will not send any data over the network, such as captured exceptions

Comment (Anakin100100) [reply config/initializers/sentry.rb:9]: No sample rate is how many requests are sampled which is different from where the data should go when an error is caught. Sentry automatically looks for the SENTRY_DSN environment variable to find where it should report errors.

Comment (micorix) [reply config/initializers/sentry.rb:9]: Ok, cool that DSN is fetched automaticly from env vars.

Can we source traces sample rate from env vars too? (We would possibly want different rates for prod and test)

Comment (Anakin100100) [reply config/initializers/sentry.rb:9]: Sure, we don't really want to trace anything in test and development environments.

**Review by micorix — APPROVED** [review]

**Pull request merged**